<launch>

   <master auto="start"/>

   <!-- Run the map server --> 
   <!-- <arg name="map" default="nav.yaml" />
   <node name="map_server" pkg="map_server" type="map_server" args="$(find tb3_navigation)/map/$(arg map)"/> -->

   <!--- Run AMCL --> 
   <!-- <include file="$(find tb3_navigation)/launch/amcl.launch" /> -->

   <include file="$(find tb3_navigation)/launch/gmapping.launch" />

   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
      <rosparam file="$(find tb3_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
      <rosparam file="$(find tb3_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
      <rosparam file="$(find tb3_navigation)/params/local_costmap_params.yaml" command="load" />
      <rosparam file="$(find tb3_navigation)/params/global_costmap_params.yaml" command="load" /> 
      <rosparam file="$(find tb3_navigation)/params/base_local_planner_params.yaml" command="load" />
   </node>

   <node pkg="rviz" type="rviz" name="rviz" />
   <!-- <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher" />
   <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" /> -->

</launch> 
